src/construction.cpp

```diff
@@ -1275,7 +1275,7 @@ void load_construction( JsonObject &jo )
     con.category = jo.get_string( "category", "OTHER" );
     // constructions use different time units in json, this makes it compatible
     // with recipes/requirements, TODO: should be changed in json
-    con.time = jo.get_int( "time" ) * 1000;
+    con.time = to_moves<int>( time_duration::from_minutes( jo.get_int( "time" ) ) );
```

Comment:
Eventually this should just be a duration, but that's out-of-scope for what you're doing.